include/noise/protocol/util.h

```diff
@@ -30,6 +30,8 @@
 extern "C" {
 #endif
 
+int noise_init();
```

Comment:
I figured it should be public API because some users will initialize crypto libs (especially OpenSSL) themselves. If you like, I'll move it internal and do the pthread_once dance.